src/template/express.ts.hbs

```diff
@@ -14,12 +16,16 @@ import { {{bodyDiscriminator.name}} } from '{{bodyDiscriminator.path}}'
 {{/each}}
 {{/each}}
 
+{{#each middlewares}}
+import { {{name}} } from '{{relativePath}}'
+{{/each}}
+
 const schemas = {{json schemas}}
 
 export function bindToRouter (router: express.Router) {
   {{#each controllers}}
   {{#each methods}}
-  router.{{verb}}('{{endpoint}}'{{#each security}}, securityMiddleware({{json this}}){{/each}}, async (req: express.Request, res: express.Response, next: express.NextFunction) => {
+  router.{{verb}}('{{endpoint}}'{{#each security}}, securityMiddleware({{json this}}){{/each}}{{#each middlewares}}, {{#if this.args}}{{this.name}}({{#each this.args}}{{this}}{{#unless @last}}, {{/unless}}{{/each}}){{else}}{{this.name}}{{/if}}{{/each}}, async (req: express.Request, res: express.Response, next: express.NextFunction) => {
```

Comment:
This is getting hard to read, maybe break route arguments onto new lines in the future.

Reply:
at some point it make more sense to maybe change the templating system to directly use ts-morph to create the correct TS code via the AST